app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3Theme.kt

```diff
@@ -149,7 +151,9 @@ internal fun lightHedvigColorScheme(hedvigTonalPalette: HedvigTonalPalette) = He
   onTypeElement = hedvigTonalPalette.greyscale1000,
   typeContainer = hedvigTonalPalette.green100,
   onTypeContainer = hedvigTonalPalette.green800,
-  typeHighlight = hedvigTonalPalette.green300,
+  lightTypeContainer = hedvigTonalPalette.green50,
+  onLightTypeContainer = hedvigTonalPalette.greyscale1000,
+  typeHighlight = hedvigTonalPalette.green100,
```

Comment:
So it only looks like the differences with typeContainer/onTypeContainer are
green100 -> green50
green800 -> greyscale1000

And then there is no difference at all for dark mode.
This looks like it's super close to the existing behavior, where was it that this was needed? Also, the list here https://www.figma.com/file/qUhLjrKl98PAzHov9ilaDH/Hedvig-UI-Kit?type=design&node-id=91%3A1127&mode=design&t=EriUfdYxfXrM0HK7-1 does not seem to show those.

I am just curious if this is a case where we just got 1 place which is a bit different, and I wonder if we really need to add this overhead here for that one place. Maybe that component should follow what the rest of the design system uses as opposed to making this exception?

Reply:
Or vice, versa, if everything else needs to be adjusted to this color. That might work too if the design needs that to happen instead.

Reply:
It's button alt default here: https://www.figma.com/file/qUhLjrKl98PAzHov9ilaDH/Hedvig-UI-Kit?type=design&node-id=91-1139&mode=design&t=B7600qn0jU7A7Hxx-4 

so I'm guessing it's not one place? Like here: 
https://www.figma.com/file/WlwFCAWywXNcXVgPfYVmWu/Final-Design?type=design&node-id=7014-14100&mode=design&t=fb2WODrzjxlsu4G9-4
https://www.figma.com/file/WlwFCAWywXNcXVgPfYVmWu/Final-Design?type=design&node-id=1996-35666&mode=design&t=fb2WODrzjxlsu4G9-4
https://www.figma.com/file/WlwFCAWywXNcXVgPfYVmWu/Final-Design?type=design&node-id=9885-35843&mode=design&t=fb2WODrzjxlsu4G9-4

Maybe my dark mode is wrong though, I'm not sure here